packages/editor-ui/src/components/Modals.vue

```diff
@@ -75,10 +90,12 @@ export default Vue.extend({
 		PersonalizationModal,
 		TagsManager,
 		UpdatesPanel,
+		ValueSurvey,
 		WorkflowSettings,
 		WorkflowOpen,
 	},
 	data: () => ({
+		CONTACT_PROMPT_MODAL_KEY,
```

Comment:
thank you for keeping alphabetic sort. this is something I often struggle to remember